src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -5327,4 +5327,7 @@ nuget locals global-packages -list</value>
     <value>nuget restore MySoluion.sln</value>
     <comment>command line example, do not localize</comment>
   </data>
+  <data name="CommandDirectDownload" xml:space="preserve">
+    <value>Download directly without populating the global package cache</value>
```

Comment:
We -DirectDownload specified a nuget restore will not popular the Global Packages Folder or the v3 http cache.